source/Patches/AttackStackSequence_OnAttackComplete.cs

```diff
@@ -0,0 +1,134 @@
+﻿using BattleTech;
+using BattleTech.BinkMedia;
+using Org.BouncyCastle.Math.Raw;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CustomSalvage
+{
+    [HarmonyPatch(typeof(AttackStackSequence), "OnAttackComplete")]
+    public static class AttackStackSequence_OnAttackComplete
+    {
+        public static bool Prepare() => Control.Instance.Settings.ShowSalvageFloaties;
+
+        [HarmonyPriority(Priority.Last)]
+        public static void Prefix(AttackStackSequence __instance, MessageCenterMessage message)
+        {
+            List<AttackDirector.AttackSequence> directorSequences = __instance.directorSequences;
+            if (directorSequences != null)
```

Comment:
use early return, meaning negative and return. avoids nesting. other languages call this semantically a guard and they also ask to keep it flat